gatsby/src/components/looking-for-something/looking-for-something.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import classes from './looking-for-something.module.scss';
+import { ArrowForwardIos, Phone } from '@material-ui/icons';
+
+const LookingForSomething = () => {
+  return (
+    <div className={classes.container}>
+      <section className={classes.section}>
+        <header className={classes.header}>
+          <strong>Stále</strong> hledáte?
```

Comment:
I think it will be needed to replace by https://github.com/cesko-digital/covid.gov.cz/blob/6b1b6af6c27aaeb09a9242e327b836cb9cd3ae18/gatsby/src/components/bolded-title/index.tsx after merge